### ISSUE TITLE
Add DESTDIR support

### ISSUE THREAD

**Issue by AliceWonderMiscreations** [Bit-Wasp/secp256k1-php#34]
I will try to do this when I get home after holidays and submit a patch to makefile if it is not done by then, but DESTDIR support would make installing in a package manager build environment (e.g. RPM) a lot easier.

**Comment by AliceWonderMiscreations** [comment]
Nevermind - looked at the makefile, it does the same thing as DESTDIR but with INSTALL_ROOT

**Comment by afk11** [comment]
Are you in any way experienced with writing build scripts? I've been hoping to compile libsecp256k1 and the extension in one go, but I'm not familiar with the build system.

Reason I ask, is when libsecp256k1 is formally released, we could look at packaging a .deb via gitian builds. We'll need a copy signed by sipa et al, or build it ourselves. 

Also, I'd love to keep a copy of secp256k1 in this repository so it can compile if the library isn't available on the target system..

**Comment by AliceWonderMiscreations** [comment]
Hi - I wrote an rpm spec file for it that BuildRequires secp256k1-devel - which I also built an RPM build script for.

It works for building the module as an RPM but the module presently doesn't appear to be working - but make test works. I suspect changes in the bitcoin library.

I am not familiar with packaging .deb but for RPM, I am rather competent. I'll post the spec file if you want it.
